source/best-practices/backup.rst

```diff
@@ -43,20 +43,35 @@
 只同步改动过的文件，所需时间较短。考虑以上两点因素以及硬盘摔坏带来的精神和身体上的损失，
 我们推荐备份家目录或者至少备份家目录下重要的子目录。有需求的用户可以考虑全盘备份。
 
+建议每隔一段时间（如每周、每月）做一次备份。放假前、出差开会前，也建议备份一下。
+
 Linux
 ------
 
 rysnc
 ^^^^^^
 
-至少每周将工作电脑中的文件备份到移动硬盘中。推荐使用 ``rsync`` 命令进行备份，其用法为::
+使用 ``rsync`` 命令进行备份十分简便。假设用户名为 seismo-learn，移动硬盘下的备份目录
+为 :file:`/mnt/seismo-learn/backup/` 。使用以下命令可以将家目录下的所有子目录和文件
+完整同步到备份目录下，此时备份目录是家目录的一个镜像::
 
     $ rsync --delete -av /home/seismo-learn/ /mnt/seismo-learn/backup/
```

Comment:
Please change the command to:
```
$ rsync -av --delete /home/seismo-learn/ /mnt/seismo-learn/backup/
```

Reply:
done in 65647f06551eee8224e040a5f34f008bc52e3c17.